.travis.yml

```diff
@@ -27,3 +27,4 @@ env:
     # prevent wine popup dialogs about installing additional packages
     - WINEDLLOVERRIDES="mscoree,mshtml="
     - WINEDEBUG="-all"
+    - DEBUG=electron-packager
```

Comment:
@malept are you okay enabling this on Travis to help debug this flakiness? I haven't been successful in getting a failure in this pull request, so I'd like to enable this temporarily on master until we figure things out.

It will make the output more verbose in the Travis reporter, but should hopefully give more insights in to where/when the failures are happening.